www/js/metrics/metricsHelper.ts

```diff
@@ -68,21 +75,21 @@ export function formatDateRangeOfDays(days: DayOfMetricData[]) {
 /* formatting data form carbon footprint calculations */
 
 //modes considered on foot for carbon calculation, expandable as needed
-const ON_FOOT_MODES = ['WALKING', 'RUNNING', 'ON_FOOT'] as const;
+export const ON_FOOT_MODES = ['WALKING', 'RUNNING', 'ON_FOOT'] as const;
```

Comment:
I don't really think we need these any more. These were originally present because we thought we may want to return the `raw` section summaries. But we don't really use those any more. Even in the section summaries, we only have `cleaned` and `inferred` (and now `ble_sensed`). I think we should assume that we will not ever return raw section summaries and simplify this code.